src/engine/runner/shared/git-clone.ts

```diff
@@ -0,0 +1,14 @@
+import { resolve } from "path";
+import { execSync } from "child_process";
+
+interface GitCloneParameters {
+  url: string;
+  name: string;
+  dirname: string;
+}
+
+export const gitClone = ({ url, name, dirname }: GitCloneParameters) =>
+  execSync(`git clone ${url} ${name}`, {
+    stdio: [0, 1, 2], // we need this so node will print the command output
```

Comment:
`stdio: "inherit"` will do the job and is more understandable then array of numbers